### ISSUE TITLE
AWS ElasticTranscoder support

### ISSUE THREAD

**Issue by FunkyloverOne** [carrierwaveuploader/carrierwave#2259]
Hi guys!
My idea is to make possible for CarrierWave users to use AWS ElasticTranscoder for remote processing instead of local via ImageMagick.
I would like to help you guys with this, but first I would like to ask you is my idea of implementation is actually good at all. So, instead of extending `fog-aws` with this, I think that it's actually make more sense to extend CarrierWave first, using `aws-sdk-elastictranscoder` gem for example.
So basically I think it should be a module similar to `CarrierWave::MiniMagick`, which will be creating jobs for elastic transcoder under the hood.

P.S. But maybe all that stuff is an overhead? Maybe there actually no need for it? Is there some much easier solutions?
